src/main/java/core/basesyntax/service/OperationStrategy.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+
+public interface OperationStrategy {
+    OperationHandler get(FruitTransaction.Operation operation);
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present
```suggestion
```